extensions/amp-nexxtv-player/amp-nexxtv-player.md

```diff
@@ -88,6 +89,10 @@ Indicates the source from which the embedded domain media is played. By default
 
 Ads are enabled by default. Set value to 1 to disable.
 
+#### data-streaming-filter (optional)
```

Comment:
also document the new `data-disable-ads` ?

Reply:
@aghassemi no, disable-ads was introduced in previous release. only data-streaming-filter is new.